src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol_ImplementationChecks.cs

```diff
@@ -1025,10 +1016,95 @@ private void CheckNewModifier(Symbol symbol, bool isNew, DiagnosticBag diagnosti
                 suppressAccessors = true;
             }
 
+            return;
+
             bool isOrContainsErrorType(TypeSymbol typeSymbol)
             {
                 return (object)typeSymbol.VisitType((currentTypeSymbol, unused1, unused2) => currentTypeSymbol.IsErrorType(), (object)null) != null;
             }
+
+            static void checkValidNullableMethodOverride(
+                Location overridingMemberLocation,
+                MethodSymbol overriddenMethod,
+                MethodSymbol overridingMethod,
+                DiagnosticBag diagnostics,
+                bool checkParameters)
+            {
+                var compilation = overridingMethod.DeclaringCompilation;
+                if (overriddenMethod is null ||
+                    overridingMethod is null ||
+                    compilation is null ||
+                    ((CSharpParseOptions)overridingMemberLocation.SourceTree?.Options)
+                        ?.IsFeatureEnabled(MessageID.IDS_FeatureNullableReferenceTypes) != true)
```

Comment:
This can be `compilation.IsFeatureEnabled(MessageID.IDS_FeatureNullableReferenceTypes)` since all parse trees are checked that they have the same version.

Reply:
That is also necessary because of [https://github.com/dotnet/roslyn/issues/35374](https://github.com/dotnet/roslyn/issues/35374)

---
In reply to: [279929478](https://github.com/dotnet/roslyn/pull/34718#discussion_r279929478) [](ancestors = 279929478)